packages/rn-tester/RNTesterPods.xcodeproj/project.pbxproj

```diff
@@ -586,7 +589,58 @@
 			);
 			runOnlyForDeploymentPostprocessing = 0;
 			shellPath = /bin/sh;
-			shellScript = "export NODE_BINARY=node\nexport PROJECT_ROOT=$SRCROOT/..\nexport SOURCEMAP_FILE=../sourcemap.ios.map\n# export FORCE_BUNDLING=true\n\"$SRCROOT/../../scripts/react-native-xcode.sh\" $SRCROOT/../../packages/rn-tester/js/RNTesterApp.ios.js\n";
+			shellScript = "export NODE_BINARY=node\nexport SOURCEMAP_FILE=../sourcemap.ios.map\nexport USE_HERMES=true\nexport PROJECT_ROOT=\"$SRCROOT/../../\"\nexport HERMES_PATH=\"$SRCROOT/../../node_modules/hermes-engine\"\n\"$SRCROOT/../../scripts/react-native-xcode.sh\" $SRCROOT/../../packages/rn-tester/js/RNTesterApp.ios.js\n";
```

Comment:
Is this going to require people manually modify their Xcode project from Xcode? Is it time to pull this into a separate script file to make upgrading easier?

Reply:
Nope. Forgot to remove `USE_HERMES` after this commit https://github.com/facebook/react-native/pull/29914/commits/0f73afb8c7b700878fd8dbd0a227f4d27fbb6a9a. Good catch!